.github/workflows/ci.yaml

```diff
@@ -17,9 +17,29 @@ jobs:
       - run: make install-dev-tools
       # Make asdf aware of globally installed binaries
       - run: asdf reshim
-
-      - run: make test
       - run: make lint
+  tests:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: asdf_install
+        uses: asdf-vm/actions/install@v1
+      - run: make install-dev-tools
+      # Make asdf aware of globally installed binaries
+      - run: asdf reshim
+      - run: make test
+  e2e-tests:
```

Comment:
I'd move this to a separate yml file, so that it can run in parallel

Reply:
I believe that's how it works currently 
![image](https://user-images.githubusercontent.com/6951209/219600706-026c64fc-518a-4de2-81d6-ba83fc469475.png)
https://docs.github.com/en/actions/using-jobs/using-jobs-in-a-workflow